source/annotations/annotations.go

```diff
@@ -18,37 +18,39 @@ import (
 )
 
 const (
+	AnnotationKeyPrefix = "external-dns.alpha.kubernetes.io/"
```

Comment:
Any chance we move this changes to it's own PR?

Reply:
Extracted into #5621